tensorflow/compiler/aot/tfcompile_main.cc

```diff
@@ -80,14 +81,24 @@ Status Main(const MainFlags& flags) {
     return Status::OK();
   }
 
-  // Read and initialize the graph.
-  if (flags.graph.empty()) {
-    return errors::InvalidArgument("Must specify --graph");
+  if ((flags.graph.empty() && flags.hlo_module.empty())
+   || (!flags.graph.empty() && !flags.hlo_module.empty())) {
```

Comment:
This is just an XNOR (`flags.graph.empty() == flags.hlo_module.empty()`), but I assume clarity is more important